web_ui/src/api.ts

```diff
@@ -33,6 +33,7 @@ export interface IUsageBillingPageArgs {
   readonly teamId: string
 }
 export interface IUsageBillingPageApiResponse {
+  readonly canSubscribe: boolean
```

Comment:
has the api been updated to return this?

Reply:
fixed